gecode/plan.sh

```diff
@@ -0,0 +1,43 @@
+pkg_name=gecode
+pkg_distname=$pkg_name
+pkg_origin=core
+pkg_version=3.7.3
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_source="http://www.gecode.org/download/gecode-${pkg_version}.tar.gz"
+pkg_shasum=e7cc8bcc18b49195fef0544061bdd2e484a1240923e4e85fa39e8d6bb492854c
+pkg_dirname=${pkg_distname}-${pkg_version}
+pkg_deps=(
+  core/gcc
```

Comment:
whoops - is gcc needed for runtime as well? If so, then the line in pkg_build_deps is duplicated - only one should be needed.